### PR TITLE
create basic project

### PR DESCRIPTION
**Pull request by Faustiks**
# Description

created basic project
add gems: 
gem 'pg', '1.3.0.rc3'
gem 'rspec', '\~> 3.5'
gem 'fast_jsonapi'
gem 'rubocop', require: false
gem 'oj', '~> 3.3', '>= 3.3.5'
gem 'devise'
gem 'devise-jwt'
gem 'uuid'
gem 'rack-cors'

add models
add device\device-jwt registration\authentication

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: server/Gemfile [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+gem "devise"
+gem "devise-jwt"
+gem "fast_jsonapi"
+gem "oj", "~> 3.3", ">= 3.3.5"
+gem "pg", "1.3.0.rc3"
+gem "rack-cors"
+gem "rspec", "~> 3.5"
+gem "rubocop", require: false
+gem "rubocop-rails", "~> 2.13", ">= 2.13.2"
+gem "rubocop-rails_config"
```
Comment (bearded) [inline server/Gemfile]: Those gems have to be specified in the section `group :development, :test do` 
see line 49 in this file

File: server/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
```
Comment (bearded) [inline server/README.md:8]: would be good to mention which ruby/rails/db versions will be used in this project
and which command have to be executed to run the project

File: server/Gemfile (hunk 1 above)
Comment (bearded) [reply server/Gemfile]: please also add gems `rspec-rails` and `simplecov`

File: server/config/database.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+#       Install PostgreSQL and put its /bin directory on your path.
+#
+# Configure Using Gemfile
+# gem "pg"
+#
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  username: postgres
+  password: DgefKlipsa
```
Comment (bearded) [inline server/config/database.yml]: please use empty password, as it may cause problems for other people to connect DB

**Review by bearded — CHANGES_REQUESTED** [review]

File: server/README.md (hunk 2 above)
Comment (Faustiks) [reply server/README.md:8]: will be added later in next PR

**Review by kapatsila-mariia — APPROVED** [review]

**Review by le-murier — APPROVED** [review]

**Pull request merged**